src/averageAges.js

```diff
@@ -14,12 +14,30 @@
  *
  * @return {number}
  */
+const male = person => person.sex === 'm';
+const female = person => person.sex === 'f';
+
+const calculateAverage = (ages) => {
+  const sumOfAge = ages
+    .map(person => person.died - person.born)
+    .reduce((sum, age) => sum + age, 0) / ages.length;
+
+  return sumOfAge;
```

Comment:
name is misleading - it's not a sum, but an actual average, right?
```suggestion
  const averageAge = ages
    .map(person => person.died - person.born)
    .reduce((sum, age) => sum + age, 0) / ages.length;

  return averageAge;
```
or 
```suggestion
  const sumOfAge = ages
    .map(person => person.died - person.born)
    .reduce((sum, age) => sum + age, 0);

  return sumOfAge / ages.length;
```